libvirtnbdbackup/common.py

```diff
@@ -72,6 +73,8 @@ def setLogLevel(verbose: bool) -> int:
 def sshSession(args: Namespace, remoteHost: str) -> Union[ssh.client, None]:
     """Use ssh to copy remote files"""
     try:
+        if args.action == "restore":
```

Comment:
I'm not sure if this is the preferred way to detect the mode that should be used. However, it works without having to change much.